README.md

```diff
@@ -53,7 +53,7 @@ A generated Hyrax-based Research Data Repository application
 6. change to the new folder
 `cd /vagrant/rdr`
 
-7. Copy the role map config file `cp config/role_map.yml.sample config/role_map.yml`
+~~7. Copy the role map config file `cp config/role_map.yml.sample config/role_map.yml`~~ *no longer needed -- [see below](#3212018----changes-to-role-management)*
```

Comment:
@mikedaul This is probably correct but it would probably be good (if you haven't done so already) to verify that the app and test suite run successfully without this file.  Just want to be sure that there isn't something in the app (or lower levels) that is counting on this file being in place even if it's not actually used.

Reply:
@coblej - good point! I'll try it out and see.

Simon mentioned that when he cloned the repo the sample file was missing, so I assumed (perhaps incorrectly) we just didn't need this any longer.

Reply:
@coblej I'm running into a problem at step 10. Any idea what this is about? (seems unrelated to the role_map file)

+++

ubuntu@samvera:/vagrant/rdr$ rake db:migrate
rake aborted!
PG::ConnectionBad: FATAL:  password authentication failed for user "hydra"
FATAL:  password authentication failed for user "hydra"

Reply:
@mikedaul Have you followed the instructions at https://duldev.atlassian.net/wiki/spaces/DDR/pages/427491331/One-Time+Setup+to+Use+Postgres+instead+of+Sqlite ?  I usually associate the error that you're getting with not having done that one-time postgres setup yet.

Reply:
@coblej I completely forgot about that! I'll add the relevant steps in the readme. Thanks!

Reply:
@mikedaul The instructions were written in the context of developers who were already set up and had been using sqlite.  For someone setting up the application from scratch, some of the steps may not be necessary -- probably, the clearing out data in Fedora and Solr steps, since they will presumably be empty anyway if you're just getting things set up.  And some of the steps (e.g., the last two) may be redundant with other steps already in the README.  The step to create the default admin set should be done _after_ the postgres setup.

Reply:
@coblej Yup, I just pulled in the relevant steps and linked to the confluence page as well, for whatever that's worth. see: https://github.com/duke-libraries/rdr/tree/readme-patch

Reply:
@coblej Sorry for all the messages! Finally got the app running, seems to be safe to delete step 7 :)

Thanks for your help!